tools/codegen/gen.py

```diff
@@ -1058,8 +982,6 @@ def computeSchemaRegister() -> Dict[str, object]:
             schema_registrations = list(mapMaybe(
                 compute_type_method(None, target=Target.REGISTRATION, op_registration_whitelist=None, def_only=True),
                 native_functions))
-            # See Note [Byte-for-byte compatibility]
-            schema_registrations.sort()
```

Comment:
Removing the sort() call- not sorting actually seems more reasonable, since the functions in `Functions.cpp`, etc, are all now written in the same order as in `native_functions.yaml`